### PR TITLE
Update Expense Storage

### LINKED ISSUE #100: Update Storage for Expense Feature
Update Storage and MainApp for Expense Feature

### LINKED ISSUE #113: As a Home-based beauty salon owner, I want to add an expense
As a Home-based beauty salon owner, I want to add an expense so that I can record a new expense that I have made

### LINKED ISSUE #114: As a Home-based beauty salon owner, I want to Tag my expense
As a Home-based beauty salon owner, I want to Tag my expense so that I can organise my expenses according to the categories they fall under, such as an expense related to a particular service

### LINKED ISSUE #115: As a Home-based beauty salon owner, I want to delete an expense
As a Home-based beauty salon owner, I want to delete an expense so that I can remove an expense that I may have added mistakenly

### LINKED ISSUE #116: As a home-based beauty salon owner, I want to List my expenses
As a home-based beauty salon owner, I want to List my expenses so that I can See all the expense entries that I have

### LINKED ISSUE #117: As a Home-based beauty salon owner, I want to Edit an expense
As a Home-based beauty salon owner, I want to Edit an expense, so that I can Change the details of an expense, in the case that I input the wrong price/description of the expense

### PR DESCRIPTION
**Pull request by yanlynnnnn**
Updated Expense Storage and MainApp, able to display CommandResult on UI for all v1.2 expense commands

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/expense/AddExpenseCommandParser.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ public AddExpenseCommand parse(String args) throws ParseException {
         }
 
         Description description = ParserUtil.parseDescription(argMultimap.getValue(PREFIX_DESCRIPTION).get());
-        IsFixed isFixed = ParserUtil.parseIsFixed(argMultimap.getValue(PREFIX_ISFIXED).get());
+        IsFixed isFixed = ParserUtil.parseIsFixed(argMultimap.getValue(PREFIX_ISFIXED).get().charAt(0));
```
Comment (hanstirtaputra) [inline src/main/java/seedu/address/logic/parser/expense/AddExpenseCommandParser.java]: With this implementation of the parsing of user input, what happens if the user inputs something like:
`addexp ... f/for now no` OR  `addexp ... f/think no`. 

Although in an actual use scenario this is very unlikely to, or will never happen, but would the parser still allow the creation of the expense object in this scenario?

**Review by hanstirtaputra — COMMENTED** [review]
I think it looks good overall! I just have one question regarding the implementation of the parser for the IS_FIXED input.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -759,7 +759,7 @@ You just purchased a bottle of conditioner for your client's hair treatment toda
 You can follow the steps below to add the expense to GrAB3.
 
 Steps :
-1. Type `addexp d/conditioner f/f v/15.00 dt/28-10-2020 t/hair supplies` in to _Command Box_.
+1. Type `addexp d/conditioner f/f v/15.00 dt/28-10-2020 t/hairsupplies` in to _Command Box_.
```
Comment (galvinleow) [inline docs/UserGuide.md:762]: Cannot handle white space?

Comment (yanlynnnnn) [reply docs/UserGuide.md:762]: I'm currently using the Tag Class provided by AB3 which does not handle white space, will handle it subsequently if it is consistent with the other features using the same Tag Class.

File: src/main/java/seedu/address/logic/parser/expense/AddExpenseCommandParser.java (hunk 1 above)
Comment (yanlynnnnn) [reply src/main/java/seedu/address/logic/parser/expense/AddExpenseCommandParser.java]: Noted, I have changed it from a character to a String. Thanks for pointing out!

**Review by hanstirtaputra — APPROVED** [review]
LGTM

**Pull request merged**